lib/dialects/cockroachdb/crdb-tablecompiler.js

```diff
@@ -7,6 +7,68 @@ class TableCompiler_CRDB extends TableCompiler {
     super(client, tableBuilder);
   }
 
+  addColumns(columns, prefix, colCompilers) {
+    if (prefix === this.alterColumnsPrefix) {
```

Comment:
Can you put similar code in function in pg-tablecompiler that add queries for alter columns, call the function in pg and crdb file, then make cockroachdb specific pushQuery ? This code is duplicated.